package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "test": "npm test",
     "start": "webpack-cli --config ./webpack.config.js --mode development --watch",
-    "start:dev": "webpack-dev-server --config ./webpack.config.js --hot --inline --mode development --color",
+    "start:dev": "webpack-dev-server --config ./webpack.config.js --host 192.168.7.125 --hot --inline --mode development --color",
```

Comment:
Is this needed for same-origin or something?

Reply:
Oh, hmm. We need to generalize this so that it connects to the simulator bridge. Would localhost or 0.0.0.0 just work here?

Reply:
I bet localhost would work...

Reply:
Oh you mean for the simulator to connect to the bridge right? Have you tried it without any `--host`? I think it might just work.